pkgs/tools/misc/envdir-go/default.nix

```diff
@@ -0,0 +1,29 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  version = "1.0.0";
+  name = "envdir-${version}";
+
+  goPackagePath = "github.com/d10n/envdir";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "d10n";
+    repo = "envdir";
+    sha256 = "1wdlblj127skgynf9amk7waabc3abbyxys9dvyc6c72zpcpdy5nc";
+  };
+
+  goDeps = ./deps.nix;
+
+  preBuild = ''
+   # TODO: is there a way to get the commit ref so we can set main.buildCommit?
```

Comment:
No, the release tarball does not contain the ref. It would be helpful to open an issue to the upstream author to actually set the `COMMIT` in `Makefile` to the corresponding ref when generating the release tarball.